README.md

```diff
@@ -124,13 +124,13 @@ export He_Cosmos_Col=movies
 az cosmosdb create -g $He_Cosmos_RG -n $He_Name
 
 # create the database
-az cosmosdb database create -d $He_Cosmos_DB -g $He_Cosmos_RG -n $He_Name
+az cosmosdb sql database create --account-name $He_Name --name $He_Cosmos_DB --resource-group $He_Cosmos_RG
 
```

Comment:
To keep consistent throughout, change this one to use the abbreviated args too (i.e. -a, -n, -g).